dallinger/command_line/docker_ssh.py

```diff
@@ -567,13 +567,9 @@ def destroy(server, app):
     ssh_user = server_info.get("user")
     executor = Executor(ssh_host, user=ssh_user, app=app)
     # Remove the caddy configuration file and reload caddy config
-    try:
-        executor.run(f"ls ~/dallinger/caddy.d/{app}")
-    except ExecuteException:
-        print(f"App {app} not found on server {server}")
-        raise click.Abort
-    executor.run(f"rm ~/dallinger/caddy.d/{app}")
+    executor.run(f"rm -f ~/dallinger/caddy.d/{app}")
```

Comment:
Actually I do find it a bit suboptimal that we display "App removed" even in the case where the app doesn't exist at all. Maybe we could add an initial check that looks to see whether either the caddy file or the docker-compose.yml file exists, and warns if not?

Reply:
Good point. Fixed!